src/field_summary.cpp

```diff
@@ -115,15 +115,15 @@ void field_summary(global_variables &globals, parallel_ &parallel) {
 					           result.buffer);
 				},
 				[](const ctx &ctx, id<1> lidx) { ctx.local[lidx] = {}; },
-				[ymax, ymin, xmax, xmin](ctx ctx, id<1> lidx, id<1> idx) {
+				[ymin, xmax, xmin](const ctx &ctx, id<1> lidx, id<1> idx) {
```

Comment:
Is this strictly `const` when `ctx.actual.*` is written?

Reply:
It's a const reference so it only prevents reassignment of the variable itself, but that's better than nothing; it matches every other use of lambdas in the a reduction call